### PR TITLE
Pbi 628 organ loading bug

### PR DESCRIPTION
**Pull request by edlu77**
#628

### CODE REVIEW

File: projects/ccf-rui/src/app/core/store/model/model.state.ts [hunk 1]
```diff
@@ -163,12 +163,8 @@ export class ModelState extends NgxsImmutableDataRepository<ModelStateModel> {
     super.ngxsOnInit();
 
     this.referenceData = this.injector.get(ReferenceDataState);
-  }
-
-  ngxsAfterBootstrap(): void {
-    super.ngxsAfterBootstrap();
 
-    if (this.globalConfig.organ) {
+    if (this.globalConfig.organ && this.globalConfig.embedded) {
```
Comment (bherr2) [inline projects/ccf-rui/src/app/core/store/model/model.state.ts:167]: Why is this needed? Why should this only be in embedded mode?

Comment (edlu77) [reply projects/ccf-rui/src/app/core/store/model/model.state.ts:167]: The default organ should only show up in embedded mode. Otherwise the user selects the starting organ from the registration modal.

Comment (bherr2) [reply projects/ccf-rui/src/app/core/store/model/model.state.ts:167]: It's not the default organ. When specified as such it is _the_ organ to load. Why does that only have to happen in embedded mode? Guess we need to revisit what embedded is used for. Is this change key for fixing the organ loading bug?

File: projects/ccf-rui/src/app/core/store/icon-registry/icon-registry.state.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export class IconRegistryState extends NgxsDataRepository<void> {
       const baseHref = globalConfig.baseHref || '';
       const safeDef: IconDefinition = {
         name, namespace,
-        url: url && sanitizer.bypassSecurityTrustResourceUrl(baseHref + url),
+        url: url && sanitizer.bypassSecurityTrustResourceUrl(url),
```
[the remark this answers is not in the document]
Comment (edlu77) [reply projects/ccf-rui/src/app/core/store/icon-registry/icon-registry.state.ts:60]: It was breaking the organ selection while in embedded mode. I'm not sure why.

[the remark this answers is not in the document]
Comment (bherr2) [reply projects/ccf-rui/src/app/core/store/icon-registry/icon-registry.state.ts:60]: Ok, let's talk about it tomorrow. baseHref is critical for embedding the rui in external sites (especially those that use their own base href in their application).

File: projects/ccf-rui/src/app/core/store/model/model.state.ts (hunk 1 above)
Comment (edlu77) [reply projects/ccf-rui/src/app/core/store/model/model.state.ts:167]: Okay I guess I misunderstood the problem. The organs will load without this change but it only works when I take out baseHref. Let's talk about it tomorrow.

**Comment by bherr2** [comment]
It appears this bug is coming from a different place. Closing this.
